### ISSUE TITLE
batch_chat multi-image

### ISSUE THREAD

**Issue by MINJIK01** [OpenGVLab/InternVL#858]
Hello,  thank you for your wonderful work.

I am just wondering if there is any other way to use batch inference with multiple images with samples other than the pipeline method.

The below code shows me how to run a batch inference with a single image. I want to keep this format but modify the way to different images per question. Is there any way to batch_chat for multi-image batch inference? Or must I use the pipeline function for multi-image batch inference?

[Relevant Code]
# batch inference, single image per sample (单图批处理)
pixel_values1 = load_image('./examples/image1.jpg', max_num=12).to(torch.bfloat16).cuda()
pixel_values2 = load_image('./examples/image2.jpg', max_num=12).to(torch.bfloat16).cuda()
num_patches_list = [pixel_values1.size(0), pixel_values2.size(0)]
pixel_values = torch.cat((pixel_values1, pixel_values2), dim=0)

questions = ['<image>\nDescribe the image in detail.'] * len(num_patches_list)
responses = model.batch_chat(tokenizer, pixel_values,
                             num_patches_list=num_patches_list,
                             questions=questions,
                             generation_config=generation_config)
for question, response in zip(questions, responses):
    print(f'User: {question}\nAssistant: {response}')

**Comment by lll2343** [comment]
Hi,

what is pipeline function.

For inference, LMDeploy is recommended. You can check out the [Batch Prompts Inference](https://internvl.readthedocs.io/en/latest/internvl2.5/deployment.html#batch-prompts-inference) section for more details.

**Comment by MINJIK01** [comment]
Thank you for your reply. My intention was not to use LMDeploy, but it's ok..!

I will try ti use it.

Thank you :)
